### ISSUE TITLE
exoVAST Talk Proposal: Toy Coronagraph

### ISSUE THREAD

**Issue by dreamjade** [vast-seminars/talk-proposals#26]
### Name of software library or topic
Toy Coronagraph

### URL to software repository
https://github.com/dreamjade/Toy_Coronagraph

### Name of person who will give the talk
[Lin, Yu-Chia](https://dreamjade.github.io/yuchia/)

### Brief abstract
I present Toy Coronagraph, an open-source Python package for simulating vortex coronagraphic observations of exoplanets in exozodiacal dust. The package includes pre-defined coronagraph designs and tools for visualizing the results of simulations. Toy Coronagraph can be used to learn about coronagraphs, simulate the performance of coronagraph designs, and design new coronagraphs. We will describe the features of the package and demonstrate a series of potential use cases, ranging from a simple SNR calculation to calculating the contrast ratio for exoplanets with a range of separations and magnitudes. We will also discuss the challenges of coronagraphic imaging and the future of exoplanet imaging.

**Comment by jrenaud90** [comment]
Hey @dreamjade would you be available to give a exoVAST talk on any of the following dates?
- April 3rd
- May 1st
- June 5th

all at 11am US Eastern time? Thank you!

**Comment by dreamjade** [comment]
May 1st sounds good to me

**Comment by jrenaud90** [comment]
That sounds great! I will send along more details as we get closer.

**Comment by jrenaud90** [comment]
Hey @dreamjade - Excited for your talk in May. I wanted to provide you with a bit more information on the format of exoVAST talks and ask for a bit more info from you.

Format: Ideally your talk will be around 45 minutes with ~50% an introduction as to what your resource does and why people should use it. The other ~50% should be a walk-through, demo, or tutorial on _how_ to use your resource. The rest of the hour will be dedicated to questions and discussion. One of the goals of exoVAST is to create tutorial/informational videos for tools hosted on [EMAC](https://emac.gsfc.nasa.gov?cid=2403-001) that may not already have one (or have an old one). We plan to upload these talks to YouTube and link them to your resource on EMAC so that potential users can get an introduction. I know ~50% of 45 minutes is not enough time to give a comprehensive tutorial on your resource, but it could just be how to run one small aspect of it, how to install it, anything that you think may: 1) help other scientists be able to use your tool or 2) get them excited to check it out and learn more. I know live demos can be fraught with problems, so you are welcome to simply have images on a slide show walking through a demo rather than actually running any code.

I know this is a non-traditional format so I would be happy to talk to you more about it! If you are game, then I just need the information found at the end of this message (and which dates you could speak on).

Thank you,
Joe

~exoVAST Speaker Form
Speaker Name and Affiliation:
Speaker Pronouns:
Short Speaker Bio:
Talk Title:
You are okay with the talk being recorded and uploaded to the VAST/EMAC YouTube:
~

**Comment by dreamjade** [comment]
Speaker Name: [Yu-Chia Lin](https://dreamjade.github.io/yuchia/#about)
Affiliation: University of Arizona, [University of Arizona Space Astrophysics Lab](https://uasal.github.io/)
Speaker Pronouns: She/Her
You are okay with the talk being recorded and uploaded to the VAST/EMAC YouTube: OK
For the talk title and the Bio, I will try to provide them in a week

**Comment by dreamjade** [comment]
Talk Title:
Toy Model for Studying Exozodiacal Dust Impact

Bio & Abstract:
Join Yu-Chia, a Physics Ph.D. candidate at the University of Arizona, as she presents her insightful talk, “Toy Model for Studying Exozodiacal Dust Impact.” With a rich academic background, including a Master in Physics from The University of New Mexico and double Bachelor’s degrees in Electrical Engineering and Physics from National Taiwan University, Yu-Chia brings a unique perspective on this topic.

Her work employs a simulation-based approach to assess the impact of exozodiacal dust within the inner working angle of a coronagraph. She introduces a video-based method to identify the impact, providing a direct way to evaluate its effect on direct imaging observations.

The module has developed an optimized process for simulating face-on and circular symmetric disks, the so-called "Toy Model". This assumption significantly reduces computational workload and facilitates the generation of accurate images in mere seconds. Her work has potential applications for evaluating the science return of future missions like the Nancy Grace Roman Space Telescope and HWO.

In conclusion, Yu-Chia’s research underscores the importance of considering exozodiacal dust in direct imaging studies. She proposes a valuable tool for assessing its effects, paving the way for future exoplanetary explorations. Attendees of her talk can expect to gain a deeper understanding of these challenges and the methods to estimate them.

**Comment by jrenaud90** [comment]
Hi, @dreamjade! Looking forward to your talk this week. I just sent out a calendar invite that has the virtual zoom connection information. I will plan to join the zoom a few minutes before 11am (US Eastern) if you want to test out your slides. Let me know if you have any questions. 

Also, have you considered submitting your code to https://emac.gsfc.nasa.gov/submissions/ ? I would be happy to help you do that if you are interested.

Cheers,
Joe
